spec.go

```diff
@@ -15,6 +15,10 @@ type Spec struct {
 	Hostname string `json:"hostname"`
 	// Mounts profile configuration for adding mounts to the container's filesystem.
 	Mounts []Mount `json:"mounts"`
+	// Prestart is a list of commands to be run before the container process is executed.
+	Prestart []Command `json:"prestart"`
```

Comment:
These two fields should be embeded in a `Hooks` struct with the `json:"hooks"` tag